src/renderer/root-frame.tsx

```diff
@@ -55,11 +58,10 @@ export class LensApp extends React.Component {
 
     registerIpcListeners();
 
-    window.onbeforeunload = () => {
-      logger.info("[App]: Unload app");
-
+    window.addEventListener("beforeunload", () => {
+      logger.info(`${RootFrame.logPrefix} Unload app`);
```

Comment:
```suggestion
      logger.info(`${RootFrame.logPrefix} Unload root frame`);
```
?

Reply:
I don't think that is necessary given that `logPrefix` is already `"[ROOT-FRAME]:"`